app/containers/HomeContainer/index.js

```diff
@@ -37,89 +46,112 @@ const Container = styled.div`
     padding: ${props => props.padding}px;
   }
 `;
-const RightContent = styled.div`
-  display: flex;
-  align-self: flex-end;
-`;
+
 export function HomeContainer({
-  dispatchGithubRepos,
-  dispatchClearGithubRepos,
+  dispatchSongs,
+  dispatchClearSongs,
   intl,
-  reposData = {},
-  reposError = null,
-  repoName,
+  itunesData,
+  itunesError,
+  artistName,
   maxwidth,
   padding
 }) {
   useInjectSaga({ key: 'homeContainer', saga });
   const [loading, setLoading] = useState(false);
+  const [isPlaying, setIsPlaying] = useState(false);
```

Comment:
do we need isPlaying anymore? if needed we can use properties on `audio` to verify if something is playing